model/upload-key.go

```diff
@@ -2,11 +2,24 @@ package model
 
 import "gorm.io/gorm"
 
+type VideoType string
+
+const (
+	VideoTypeCombined     VideoType = "COMB"
+	VideoTypePresentation           = "PRES"
+	VideoTypeCamera                 = "CAM"
+)
+
+func (v VideoType) Valid() bool {
+	return v == VideoTypeCombined || v == VideoTypePresentation || v == VideoTypeCamera
+}
+
```

Comment:
Nice way of doing this! Unsure about this: did you check whether the stream.go model already has a similar enum?

Reply:
Searched for such an enum, but I did not find any :(